force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +101,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String evenOdd = null;
+        //I think the issue is with how it is calculating the double variable. I need to make sure this is giving a decimal. This code is in the apex files. try math.mod
+        Double fullResult = (double)a / 2;
+        Integer roundResult = a / 2;
+        if (fullResult > roundResult) {
+            evenOdd = 'Odd';
+        }
+        if (fullResult == roundResult) {
+            evenOdd = 'Even';
+        }
```

Comment:
Because you used a ternary earlier I will share my solution with you. 

` public static String checkEvenOdd(Integer a) {
        if(a == null ){
            return null;
        }
        return Math.mod(a, 2) == 0 ? 'Even' : 'Odd';
    }`